account_lock_date_update/wizards/account_update_lock_date.xml

```diff
@@ -34,13 +30,13 @@
         </field>
     </record>
     <record id="account_update_lock_date_act_window" model="ir.actions.act_window">
-        <field name="name">Update accounting lock dates</field>
+        <field name="name">Lock Dates</field>
```

Comment:
If you rename this, at least keep the update part:

```suggestion
        <field name="name">Update Lock Dates</field>
```

Reply:
In fact, the wizard can also be used to view the Lock Dates, because, in v14 Community, there is no other way to view the value of the Lock Dates.

Reply:
Then call it `See/Update Lock Dates`, but I don't think calling it only `Lock Dates` would be good. And definitively not in the middle of the life of a version. If you want to do such change, do it for v15.

Reply:
Please take into account my comment